tests/covalent_tests/executor/executor_plugins/local_test.py

```diff
@@ -141,26 +137,37 @@ def test_local_executor_run_exception_handling(mocker):
     kwargs = {}
     task_metadata = {"dispatch_id": "asdf", "node_id": 1}
     with pytest.raises(TaskRuntimeError) as ex:
-        le.run(_local_executor_run_exception_handling_mock_function, args, kwargs, task_metadata)
-    assert "f output" in le._task_stdout.getvalue()
+        le.run(local_executor_run_exception_handling__mock_task, args, kwargs, task_metadata)
+    le._task_stdout.getvalue() == "f output"
     assert "RuntimeError" in le._task_stderr.getvalue()
 
 
+def local_wrapper_fn_exception_handling__mock_task(x):
+    raise RuntimeError("Err")
+
+
+def local_wrapper_fn_exception_handling__mock_task(x):
+    raise RuntimeError("Err")
+
+
+@pytest.mark.skip(reason="Unable to mock traceback in subprocess")
 def test_local_wrapper_fn_exception_handling(mocker):
     import multiprocessing as mp
 
     from covalent.executor.utils.wrappers import local_wrapper
 
     args = [5]
     kwargs = {}
+    error_msg = "task failed"
     q = mp.Queue()
+    mocker.patch("traceback.TracebackException.from_exception", return_value=error_msg)
     p = mp.Process(
         target=local_wrapper,
-        args=(_test_local_wrapper_fn_exception_handling_mock_function, args, kwargs, q),
+        args=(local_wrapper_fn_exception_handling__mock_task, args, kwargs, q),
```

Comment:
Hi @AlejandroEsquivel , this test and its associated functions defined above (`local_wrapper_fn_exception_handling_mock_task`) can be deleted as there is no more `local_wrapper`. The LocalExecutor is now quite similar to DaskExecutor.